includes/Admin/Admin_Page.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Admin\Admin_Page
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Admin;
+
+/**
+ * Class is handling admin tools page functionality.
+ *
+ * @since n.e.x.t
+ */
+class Admin_Page {
+
+	/**
+	 * Initialize hooks.
```

Comment:
Please remember that all methods and function descriptions should start with a third-person verb. Also, every method and function should have a `@since n.e.x.t` annotation.
```suggestion
	 * Initializes hooks.
	 *
	 * @since n.e.x.t
```

Reply:
I have updated this now, thank you!